.github/workflows/pr_ci.yml

```diff
@@ -30,13 +30,18 @@ jobs:
       uses: actions/setup-dotnet@v1
       with:
         dotnet-version: 5.0.*
+    - name: Setup .NET Core 6.0 on ${{ matrix.os }}
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 6.0.100-rc.1.21458.32
     - name: Build
       run: |
-        dotnet build --configuration Release --framework net5.0
-        dotnet build --configuration Release --framework netcoreapp3.1
+        dotnet build --configuration Release --framework net6.0 /p:DefaultTargetFramework=net6.0
+        dotnet build --configuration Release --framework net5.0 /p:DefaultTargetFramework=net5.0
```

Comment:
So what happens if e.g. we do

    dotnet build --configuration Release --framework net6.0 /p:DefaultTargetFramework=net5.0